lib/IRGen/GenType.cpp

```diff
@@ -1125,14 +1125,19 @@ CanType TypeConverter::getExemplarType(CanType contextTy) {
   // FIXME: A generic SILFunctionType should not contain any nondependent
```

Comment:
In fact a generic silfunctiontype should not contain archetypes at all. I suspect this entire block of code is dead